lib/restaurant.rb

```diff
@@ -0,0 +1,13 @@
+class Restaurant
+
+  def take_order(order, payment, person)
+
+    if payment < order.total
+      puts "not enough grade"
+    else
```

Comment:
You don't need to write an interface - not part of the brief. 
Do not handle errors with a string - raise an error instead. 
Do not use puts except when debugging.